Annotato/Annotato/ViewController.swift

```diff
@@ -0,0 +1,181 @@
+
+        documentViewButton.center = .init(x: 100, y: 100)
+        documentViewButton.addTarget(self, action: #selector(didTapDocumentButton), for: .touchUpInside)
+        view.addSubview(documentViewButton)
+    }
+
+    private func addPdfView() {
+        view.addSubview(pdfView)
+        guard let url = Bundle.main.url(forResource: "Lab01Qns", withExtension: "pdf") else {
+            print("No such pdf")
+            return
+        }
+        guard let document = PDFDocument(url: url) else {
+            print("No such document")
+            return
+        }
+        pdfView.document = document
+        pdfView.autoScales = true
+        pdfView.delegate = self
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        pdfView.frame = view.bounds
+    }
+
+    @IBAction private func handleNextButtonTapped(_ sender: Any) {
+        pdfView.goToNextPage(nil)
+    }
+
+    private func addObservers() {
+        NotificationCenter.default.addObserver(self, selector: #selector(handlePageChange(notification:)),
+                                               name: Notification.Name.PDFViewPageChanged, object: nil)
+    }
+
+    @objc private func handlePageChange(notification: Notification) {
+        print("changed pages")
+        nextButton.isEnabled = pdfView.canGoToNextPage
+    }
+
+    private func addAnnotationAtLocation(location: CGPoint, text: String) {
+        print("add annotation at location")
+        guard let currentPage = pdfView.currentPage else {
+            return
+        }
+        let textAnnotation = PDFAnnotation(
+            bounds: CGRect(x: location.x, y: location.y, width: 20, height: 20),
+            forType: .text, withProperties: nil
+        )
+        textAnnotation.contents = text
+        textAnnotation.color = .clear
+        currentPage.addAnnotation(textAnnotation)
+    }
+
+    @IBAction private func importFiles(_ sender: Any) {
+        print("importing files")
+        let documentPicker = UIDocumentPickerViewController(forOpeningContentTypes: [UTType.pdf], asCopy: true)
+        documentPicker.delegate = self
+        documentPicker.allowsMultipleSelection = false
+        documentPicker.modalPresentationStyle = .fullScreen
+        present(documentPicker, animated: true, completion: nil)
+    }
+
+    private func addTripleTapGestureAddAnnotation() {
+        guard let pdfDefaultGestures = pdfView.gestureRecognizers else {
+            print("no gestures in pdfview")
+            return
+        }
+        for gesture in pdfDefaultGestures where gesture is UITapGestureRecognizer {
+            if let gesture = gesture as? UITapGestureRecognizer {
+                self.pdfTapGestures.append(gesture)
+            }
+        }
+        let tripleTapGestureRecognizer = UITapGestureRecognizer(
+            target: self, action: #selector(didTripleTap))
+        tripleTapGestureRecognizer.numberOfTapsRequired = 3
```

Comment:
Is the triple tap supposed to add PDFAnnotation? I don't see anything and I also don't really get what's going on.